smoke-tests/portland/router-config.json

```diff
@@ -3,20 +3,19 @@
   "updaters": [
     {
       "type": "real-time-alerts",
-      "frequencySec": 60,
       "earlyStartSec": 864000,
       "url": "https://trimet.org/transweb/ws/V1/FeedSpecAlerts/includeFuture/true/suppressSystemWideAlerts/true/",
       "feedId": "TriMet"
     },
     {
       "type": "stop-time-updater",
-      "frequencySec": 30,
+      "frequency": "30s",
```

Comment:
I've been seeing this diff all over the place. Is this required now?

Reply:
Not required but that's the new syntax. Default is 1 minute.